src/client/components/login-form.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: "",
+      password: "",
+      result: ""
+    };
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    });
+  };
+  handleSubmit = event => {
+    event.preventDefault();
+
+    fetch("http://localhost:3000/api/auth/login/ ", {
+      method: "POST",
+      headers: {
+        Accept: "application/json text/plain",
+        "Content-Type": "application/json"
+      },
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.password
+      })
+    })
+      .then(response => response.json())
+      .then(data => {
+        localStorage.setItem("token", data.token); //storing token into local storage
+        //console.log(data);
+      });
+  };
+  catch(err) {
```

Comment:
What is this catch doing? should that have a `try{}`wrapping the API call? :)

Reply:
resolved it.